### PR TITLE
Add label for User Group Big Data

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/community/pull/3304

/assign @cblecker 
cc @erikerlandson

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -593,6 +593,12 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - color: d2b48c
+      description: Categorizes an issue or PR as relevant to ug-big-data.
+      name: ug/big-data
+      target: both
+      prowPlugin: label
+      addedBy: anyone
```
Comment (cblecker) [inline label_sync/labels.yaml]: Can you add a previously for `sig/big-data` to this?

Comment (nikhita) [reply label_sync/labels.yaml]: Done. Also added `deleteAfter` for `sig/big-data`.

**Comment by erikerlandson** [comment]
lgtm (non-binding)

File: label_sync/labels.yaml (hunk 1 above)
Comment (cblecker) [reply label_sync/labels.yaml]: Actually, just remove the whole label section for sig/big-data. We don't want to delete the label. It'll be auto migrated.

**Pull request merged**